smart-contracts/contracts/PublicLock.sol

```diff
@@ -2,6 +2,7 @@ pragma solidity 0.5.12;
 
 import './interfaces/IERC721.sol';
 import './interfaces/IERC721Enumerable.sol';
+import './interfaces/IPublicLock.sol';
```

Comment:
Need to add this to the `is` list as well ;)

Reply:
Doh! Sorry, that's what happens when I'm trying to work while cooking dinner and helping kids with homework. Will fix and resubmit.